scripts/shared/lib/utils

```diff
@@ -105,12 +105,27 @@ function declare_cidrs() {
     # shellcheck disable=SC2034 # these variables are used elsewhere
     declare -gA cluster_CIDRs service_CIDRs global_CIDRs
 
-    for i in {1..3}; do
-        add_cluster_cidrs $i
+    n=${1:-3}
+    for i in $(eval echo "{1..$n}"); do
+        add_cluster_cidrs "$i"
     done
 }
 
 function declare_kubeconfig() {
+    n=$(kind get clusters | wc -l)
     export KUBECONFIG
-    KUBECONFIG=$(echo "${KUBECONFIGS_DIR}"/kind-config-cluster{1..3} | sed 's/ /:/g')
+    KUBECONFIG=$(eval echo "${KUBECONFIGS_DIR}/kind-config-cluster{1..$n}" | sed 's/ /:/g')
+}
+
+function print_clusters_message() {
+    num_clusters=$(kind get clusters | wc -l)
```

Comment:
This should be the actual cluster names

Reply:
You mean like this `clusters=$(kind get clusters)`? But then how do I get it to expand with a range on line L125?